### PR TITLE
Logo cloud

### PR DESCRIPTION
**Pull request by nikitin2009**
Task: https://leximaio.teamwork.com/#/tasks/28996950

### CODE REVIEW

**Comment by olafghanizadeh** [comment]
You can copy most of the markup, but that's just for you to have the boilerplate. You will have to modify the css etc, as we don't want changes in the forked component to affect this one.

**Comment by nikitin2009** [comment]
OK, got it

**Comment by olafghanizadeh** [comment]
This looks mostly good, but it seems like on desktop it is making the icons smaller as you add more

**Comment by nikitin2009** [comment]
Also, I noticed you made a force-push to the branch. What was that? I can't push now my changes. Should I pull that changes?

File: sections/logo-cloud.liquid [hunk 1]
```diff
@@ -17,33 +17,35 @@
 <div class="logo-cloud">
   <div class="section-{{ section.id }}-padding isolate">
 
-    <ul class="logo-cloud-list">
-      {%- for block in section.blocks -%}
+    <scrollable-slider>
+      <ul class="logo-cloud-list">
+        {%- for block in section.blocks -%}
```
Comment (olafghanizadeh) [inline sections/logo-cloud.liquid:23]: Could you add the 
`   {%- case block.type -%}` and  ` {%- when 'logo'-%}`

**Comment by olafghanizadeh** [comment]
Could you add cursor styles for the different states? You can use grab on hover and grabbing when dragging.

Maybe draggable-slider is a better name for the custom element as well?

**Pull request merged**